doc/source/theory/FlavorSpace.rst

```diff
@@ -75,30 +76,125 @@ The mapping between the Evolution Basis and the +/- Basis is given by
 - this basis is *not* normalized with respect to the canonical Flavor Basis
 - the basis transformation from the Flavor Basis is implemented in
   :data:`~eko.basis_rotation.rotate_flavor_to_evolution`
+- the photon is just a spectator and does not couple to anyone
 
-Intrinsic Evolution Bases
--------------------------
+Intrinsic QCD Evolution Bases
+-----------------------------
 
-However, the Evolution Basis is not yet the most decoupled basis if we consider intrinsic evolution.
+However, the QCD Evolution Basis is not yet the most decoupled basis if we consider intrinsic evolution.
 The intrinsic distributions do *not* participate in the |DGLAP| equation but instead evolve with a unity operator:
 this makes, e.g. :math:`T_{15}` a composite object in a evolution range below the charm mass.
 Instead, we will keep the non participating distributions here in their :math:`q^\pm` representation.
-The Intrinsic Evolution Bases will explicitly depend on the number of light flavors :math:`n_f`.
+The Intrinsic QCD Evolution Bases will explicitly depend on the number of light flavors :math:`n_f`.
 For :math:`n_f=3` we define (the other cases are defined analogously):
 
 .. math ::
-    \mathcal F \sim  \mathcal F_{iev,3} = \span(g, \Sigma_{(3)}, V_{(3)}, V_3, V_8, T_3, T_8, c^+, c^-, b^+, b^-, t^+, t^-)
+  \Sigma_{(3)} &= u^+ + d^+ +s^+\\
+  V_{(3)} = u^- + d^- + s^-\\
+  \mathcal F \sim  \mathcal F_{iev,3} &= \span(\gamma, g, \Sigma_{(3)}, V_{(3)}, V_3, V_8, T_3, T_8, c^+, c^-, b^+, b^-, t^+, t^-)
 
-where we defined :math:`\Sigma_{(3)} = \sum\limits_{j=1}^3 q_j^+` and :math:`V_{(3)} = \sum\limits_{j=1}^3 q_j^-`
-(not to be confused with the usual :math:`V_3`).
+where :math:`V_{(3)}` is to be confused with the usual (QCD like) :math:`V_3`.
```

Comment:
I would leave the def of `\Sigma_{(3)}` and `V_{(3)}` explicit, there is a missing not...

Reply:
I perfectly agree with @giacomomagni